src/store/actions/index.js

```diff
@@ -100,8 +100,9 @@ export const postVideo = (token, video, history) => (dispatch) => {
       setTimeout(() => {
         dispatch({
           type: POST_VIDEO_SUCCESS,
+          payload: res.data,
         });
-        history.push(`/video/${res.data.videoId}`);
+        history.push(`/videos/${res.data.videoId}`);
```

Comment:
I was definitely staring at the screen too long to overlook this. Thank you for the second pair of eyes Navid.